inc/functions_post.php

```diff
@@ -1021,3 +1021,28 @@ function get_post_attachments($id, &$post)
 		}
 	}
 }
+
+/**
+ * Returns bytes count from human readable string
+ * Used to parse ini_get human-readable values to int
+ *
+ * @param string $val Human-readable value
+ */
+function return_bytes($val) {
+    $val = trim($val);
+    if ($val == "")
+    	return 0;
```

Comment:
Use curly braces....
Also this check is not required if a default is set in switch-case.

Reply:
Without this condition, given empty string it would attempt to get string offset -1, giving php notice.
The function is copy-pasted from here: http://php.net/manual/en/function.ini-get.php

Reply:
Not if you declare default value which will pass 0 always other than a match.